Prediction_Eval.ipynb

```diff
@@ -0,0 +1,464 @@
+{
+  "nbformat": 4,
```

Comment:
<img width="717" alt="image" src="https://github.com/geo-smart/swe-workflow-book/assets/45752727/fff530fe-9a7b-4180-afef-1e842a567ee9">

Missing code of merge_data and predict